rundown/static/sportsbooks.py

```diff
@@ -0,0 +1,102 @@
+"""Module for static sportsbook information used by Rundown."""
```

Comment:
same to be said about sportsbooks. These will often change and more will be added.

Reply:
This one is used in `rundown.resources.event.Event` in the `use_sportsbook_names` validator. Basically the returned `Event` object has sportsbook names instead of IDs as keys for the `lines` and `line_periods` dictionaries. Again the reasoning for this is so the user can avoid trying to remember sportsbook IDs. 

The validator method would need to be changed to allow IDs as keys in case the sportsbook name isn't present in the static list.

Reply:
Right now if a new sportsbook is added that isn't in the static list, the validator will raise a ValidationError.

Reply:
Same with the above - can we build this list dynamically?